perf_test/CMakeLists.txt

```diff
@@ -50,5 +50,97 @@ if (KokkosKernels_ENABLE_PERFTESTS)
     ADD_COMPONENT_SUBDIRECTORY(blas)
     ADD_SUBDIRECTORY(performance)
     #ADD_SUBDIRECTORY(common)
+
+    IF (KOKKOSKERNELS_HAS_TRILINOS)
+      message(FATAL_ERROR "Benchmarks are not supported when building as part of Trilinos")
+    ENDIF()
+
+    find_package(benchmark QUIET)
```

Comment:
Only search for google benchmark is it is enabled via KokkosKernels_ENABLE_BENCHMARK.

Reply:
:+1: bb8a3e662b943b9c9f84dbf5f12706985c6d30dc

Reply:
Please add the `KokkosKernels_ENABLE_BENCHMARK` option around here: https://github.com/kokkos/kokkos-kernels/blob/develop/CMakeLists.txt#L53 and docs to [BUILD.md.](https://github.com/kokkos/kokkos-kernels/blob/develop/BUILD.md)

Reply:
We didn't change the top level CMakeLists.txt because it was decided with @lucbv during one meeting with him. Our understanding it that eventually, benchmark and perfsuite need to be cleaned up or unified. We will bring this up again during the meeting with Luc in case we misunderstood.